components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3429,11 +3448,25 @@ std::string RewardsServiceImpl::GetExternalWalletType() const {
           country_codes::CountryCharsToCountryID(country.at(0), country.at(1));
 
       if (id == current_country)
-        return ledger::constant::kWalletBitflyer;
+        return true;
     }
   }
+  return false;
+}
 
+std::string RewardsServiceImpl::GetExternalWalletType() const {
```

Comment:
this should have unit tests

Reply:
added: https://github.com/brave/brave-core/pull/8566/files#diff-79a8ff13d4400b7fb9810d76aa14728acb0aded37d2dc1c414a8b00396489642R114